packages/flutter/lib/src/material/material.dart

```diff
@@ -35,6 +35,15 @@ enum MaterialType {
   button,
 
   /// A transparent piece of material that draws ink splashes and highlights.
+  ///
+  /// While the material metaphor describes child widgets as printed on the
+  /// material itself and do not hide ink effects, in practice the [Material]
+  /// widget draws child widgets on top of the ink effects.
+  /// A [Material] with type transparency can be placed on top of opaque widgets
+  /// to show ink effects on top of them.
+  ///
+  /// Since the introduction of the [Ink] widget, the [Ink] widget is the
```

Comment:
Strike "Since the introduction of the [Ink] widget,".

Maybe this can be replaced with "Where possible, prefer using the [Ink] widget to print the opaque graphics on the [Material], rather than using a transparent material." or something like that.

Reply:
Changed to "Prefer using the [Ink] widget for showing ink effects on top of opaque widgets"